app/Models/Tracer/CareerHistory.php

```
@@ -6,11 +6,27 @@
 {
     use HasFactory;
 
+    protected static function booted(): void
+    {
+        static::saved(function ($career) {
+            Cache::forget('portal_total_alumni');
+            Cache::forget('portal_welcome_alumni_data');
+            Cache::forget('portal_welcome_alumni_stats');
+        });
+
+        static::deleted(function ($career) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$career'. (undefined)

(UnusedFormalParameter)